src/loki/reach.cc

```diff
@@ -0,0 +1,186 @@
+#include "loki/reach.h"
+
+using namespace valhalla::baldr;
+
+namespace valhalla {
+namespace loki {
+
+directed_reach SimpleReach(const DirectedEdge* edge,
```

Comment:
this is the main funciton that computes the reach. ive commented the hell out of it so i wont write anything else here :)